spec/unit/ai_response_board_spec.rb

```diff
@@ -33,89 +34,89 @@
       expect(ai_response.execute(game.board)).to eq(2)
     end
 
     it 'can block player from winning, example 2' do
       game = game_gateway.fetch_game
       game.board = ['X', nil, 'X',
-               nil, 'O', nil,
-               nil, nil, nil]
+                    nil, 'O', nil,
+                    nil, nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute(game.board)).to eq(1)
     end
 
     it 'can block player from winning, example 3' do
       game = game_gateway.fetch_game
       game.board = ['X', nil, nil,
-               'X', 'O', nil,
-               nil, nil, nil]
+                    'X', 'O', nil,
+                    nil, nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(6)
     end
 
     it 'can block player from winning, example 4' do
       game = game_gateway.fetch_game
       game.board = ['X', nil, nil,
-               nil, 'O', nil,
-               'X', nil, nil]
+                    nil, 'O', nil,
+                    'X', nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(3)
     end
 
     it 'can block player from winning, example 5' do
       game = game_gateway.fetch_game
       game.board = [nil, 'X', 'X',
-               nil, 'O', nil,
-               nil, nil, nil]
+                    nil, 'O', nil,
+                    nil, nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(0)
     end
 
     it 'can block player from winning, example 6' do
       game = game_gateway.fetch_game
       game.board = [nil, nil, 'X',
-               nil, 'O', 'X',
-               nil, nil, nil]
+                    nil, 'O', 'X',
+                    nil, nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(8)
     end
 
     it 'can block player from winning, example 7' do
       game = game_gateway.fetch_game
       game.board = [nil, nil, nil,
-               'X', 'O', nil,
-               'X', nil, nil]
+                    'X', 'O', nil,
+                    'X', nil, nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(0)
     end
 
     it 'can block player from winning, example 8' do
       game = game_gateway.fetch_game
       game.board = [nil, nil, nil,
-               nil, 'O', 'X',
-               nil, nil, 'X']
-               game_gateway.update(game)
+                    nil, 'O', 'X',
+                    nil, nil, 'X']
+      game_gateway.update(game)
       expect(ai_response.execute).to eq(2)
     end
 
     it 'can block player from winning, example 9' do
       game = game_gateway.fetch_game
       game.board = [nil, nil, nil,
-               nil, 'O', nil,
-               'X', 'X', nil]
+                    nil, 'O', nil,
+                    'X', 'X', nil]
       game_gateway.update(game)
       expect(ai_response.execute).to eq(8)
     end
 
     it 'can block player from winning, example 10' do
```

Comment:
Did you write the tests first to TDD minimax? It seems a little odd to have 10 examples of blocking

Reply:
Yes that is triangulation :D